static/js/popovers.js

```diff
@@ -566,8 +566,12 @@ export function open_message_menu(message) {
         return true;
     }
 
-    $(".selected_message .actions_hover .zulip-icon-ellipsis-v-solid").trigger("click");
-    focus_first_action_popover_item();
+    const $popover_reference = $(".selected_message .actions_hover .zulip-icon-ellipsis-v-solid");
+    $popover_reference.trigger("click");
+    // Focus the popover only if it is displayed.
+    if ($popover_reference.length && $popover_reference[0]._tippy) {
+        setTimeout(focus_first_action_popover_item, 0);
+    }
```

Comment:
This would be better squashed into
`actions_popover: Use tippyjs instead of bootstrap to display popover`

Because that commit removes the `if` parallel if statement.

Can you add a comment for why the `setTimeout` is required? They're generally a code smell, since almost always they're a bug where we should be passing a callback to something that will call our function once the relevant thing is ready.

(I think if I understand it correctly, what this function should ultimately be doing is just calling the click handler passing a callback, rather than just simulating a click and thus having no way to tell Tippy that it needs us to do this focus?)

Reply:
I added a variable `message_actions_popover_keyboard_toggle` to track if we need keyboard focus and removed this code. I squashed the commit in which we move and rename it to `toggle_message_actions_menu` since that looked cleaner.